docs/conf/prague/2022/news/get-ready.rst

```diff
@@ -0,0 +1,71 @@
+:template: {{year}}/generic.html
+
+.. post:: Sept 1, 2022
+   :tags: {{year}}, {{shortcode}}-{{year}}, tickets
+
+Get Ready for the conference!
+=============================
+
+Hey everyone!
+
+We're just over a week away from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.
+
+We have a few reminders before we welcome you all next Sunday:
+
+* Ticket sales close next week, so grab yours now!
+* We have a great Welcome Wagon, which has a guide, as well as introductory tours of our online platform
+* There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prep a bit for them.
+
+Ticket sales close on **September 8th**
+---------------------------------------
+
+To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **September 8th at 11:59PM Central European Time**.
```

Comment:
```suggestion
To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **September 8th at 23:59 CEST**.
```
We're not in America anymore with that weird clock ;)
Also, CET is technically UTC+1, we're still in CEST.